composer/utils/object_store/mlflow_object_store.py

```diff
@@ -53,7 +53,6 @@ def _wrap_mlflow_exceptions(uri: str, e: Exception):
     retryable_server_codes = [
         ErrorCode.Name(code) for code in [
             DATA_LOSS,
-            INTERNAL_ERROR,
```

Comment:
why were we retrying on 500s before? @jerrychen109

Reply:
because 500s can be many things, since mlflow is wrapping all server errors as 500. some of them might be retryable, so we retry

Reply:
I think we still want to retry on those? There may be other internal errors that we would need to retry on

Reply:
hm ok that's still sus to me though